### DISCUSSION TITLE
Bypass authelia page but still default to password & duo notfication

### DISCUSSION THREAD

**Discussion by Alumni1506** [authelia/authelia#6442] [General]
Hi,

Is it possible to configure authelia such that it behaves as follows?

- Visit my self hosted service e.g. nextcloud
- User is lands on nextcloud's normal landing page
- User enters their password
- In the background a duo notification is sent to the user for 2fa

The reason for above is for apps that don't work with redirects

**Comment by james-d-elliott** [comment]
Unfortunately there is no way to technically accomplish this as there is no standard contract that does this. What could theoretically work is OpenID Connect 1.0 which has quite good support in most apps.

**Reply by Alumni1506** [reply]
thanks for the clarification
